diesel/src/sqlite/types/date_and_time/chrono.rs

```diff
@@ -174,11 +219,40 @@ mod tests {
     fn times_of_day_decode_correctly() {
         let connection = connection();
         let midnight = NaiveTime::from_hms(0, 0, 0);
-        let query = select(sql::<Time>("'00:00:00'"));
-        assert_eq!(Ok(midnight), query.get_result::<NaiveTime>(&connection));
+        let valid_midnight_formats = &[
+            "00:00",
+            "00:00:00",
+            "00:00:00.000",
+            "00:00:00.000000",
+            "00:00Z",
+            "00:00:00Z",
+            "00:00:00.000Z",
+            "00:00:00.000000Z",
+            "00:00+00:00",
+            "00:00:00+00:00",
+            "00:00:00.000+00:00",
+            "00:00:00.000000+00:00",
+            "01:00+01:00",
```

Comment:
This tests [fails](https://travis-ci.org/diesel-rs/diesel/jobs/286125994#L634) on CI